### PR TITLE
Added deployment and disaster recovery content 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added deployment and disaster recovery content

### CODE REVIEW

File: docs/HowTo/Deploy/Bootnodes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+A network must have at least one operating bootnode. To allow for continuity in the event of failure, 
+configure more than one bootnode. 
+
+We do not recommend putting bootnodes behind a load balancer. We recommend putting more bootnodes on the network itself. 
+
+The [enode](../../Concepts/Node-Keys.md#enode-url) of a bootnode is tied to the node public key and IP address. 
+To simplify recovering from complete bootnode failure: 
+
+1. Create the [node key pair](../../Concepts/Node-Keys.md) (that is, the private and public key) before starting the bootnode and put
+the private key in the `/opt/pantheon` directory.
```
Comment (MadelineMurray) [inline docs/HowTo/Deploy/Bootnodes.md]: @joshuafernandes - is this assuming you're deploying in a specific manner?  ie, the /opt/pantheon directory is not the default location for the private key when you install locally.

File: docs/HowTo/Upgrade/Upgrade-Network.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+We recommend: 
+
+* Using an orchestration method (for example, Ansible or Chef) to keep all nodes in sync with your desired configuration. 
+* Storing your configuration under version control. 
+
+## Ansible 
+
+The [Pantheon Ansible role on Galaxy](https://galaxy.ansible.com/pegasyseng/pantheon) can be used directly
+or customised to suit your needs. Upgrade by running the play. The play:
```
Comment (MadelineMurray) [inline docs/HowTo/Upgrade/Upgrade-Network.md:14]: @joshuafernandes - not sure if I have wording right here around 'the play'

Comment (joshuafernandes) [reply docs/HowTo/Upgrade/Upgrade-Network.md:14]: Maybe "Upgrade the pantheon version on nodes by running the play with the new version. See the Readme on Galaxy for details"?

File: docs/HowTo/Backup/Pantheon-Backup.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+stored under source control. 
+
+## Data Backups
+
+We recommend mounting a separate volume to store data. Use the [`--data-path`](../../Reference/Pantheon-CLI/Pantheon-CLI-Syntax.md#data-path) 
+command line option to pass the path to Pantheon. 
+
+Having some data reduces the time to synchronise a new node. Periodic backups can be performed of 
+the data directory and the data sent to your preferred backup mechanism. For example, cron job and 
+rsync, or archives to cloud such as s3.
```
Comment (joshuafernandes) [inline docs/HowTo/Backup/Pantheon-Backup.md]: suggest `or 'tar.gz' achives`

File: docs/HowTo/Deploy/Bootnodes.md (hunk 1 above)
Comment (joshuafernandes) [reply docs/HowTo/Deploy/Bootnodes.md]: I believe it is just the normal deployment process and location, just repeating the path here - could be removed tbh?

File: docs/HowTo/Upgrade/Upgrade-Network.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+1. Applies any new configuration.
+1. Starts Pantheon.
+
+## Finding Peers on Restarting 
+
+Nodes store known peers in the peer table. The peer table is not persisted to disk. When a node restarts, 
+the node connects to the specified bootnodes and discover other nodes through the peer discovery process. 
+The node continues collecting data from where it was (assuming there was no data corruption in a failure scenario). 
+
+The node may also be connected to by other nodes that still have the restarted node in their peer table.
```
Comment (joshuafernandes) [inline docs/HowTo/Upgrade/Upgrade-Network.md]: Does this read easier? " When the node was operational, prior to the restart, it communicates with other peers which save its details in their peer table. These peers may connect to the restarted node and resume activity; when this happens the node will use these peers as well to discover more peers. To ensure that the restarted node successfully rejoins the network ensure at least one operational bootnode is specified."

**Review by joshuafernandes — APPROVED** [review]
Looks really good, thanks for doing this. Left some minor optional comments in

File: docs/HowTo/Backup/Pantheon-Backup.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/HowTo/Backup/Pantheon-Backup.md]: Done

File: docs/HowTo/Deploy/Bootnodes.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/HowTo/Deploy/Bootnodes.md]: Removed

File: docs/HowTo/Upgrade/Upgrade-Network.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Upgrade/Upgrade-Network.md:14]: Done

File: docs/HowTo/Upgrade/Upgrade-Network.md (hunk 4 above)
Comment (MadelineMurray) [reply docs/HowTo/Upgrade/Upgrade-Network.md]: Updated to be similar to this

**Pull request merged**